configuration.nix

```
@@ -165,6 +165,29 @@ isMaximal: {
       motion = {
         hop.enable = true;
         leap.enable = true;
+
+        precognition = {
+          enable = true;
+          setupOpts = {
+            startVisible = true;
+            showBlankVirtLine = true;
+
+            # highlightColor - automatically set by theme
+            disabled_fts = ["startify" "alpha" "dashboard"];
+            gutterHints = {
+              gg = {
+                text = "gg";
+                prio = 2;
+              };
+            };
+            hints = {
+              Caret = {
+                text = "^";
+                prio = 10;
+              };
+            };
+          };
+        };
```

```suggestion
        precognition.enable = isMaximal;
```

`configuration.nix` is only for preset configs (nix and maximal packages). If this is some kind of an example config, consider making those values the default.

Also enable precognition only in maximal configuration, since it's likely redundant on the default `nix` package.

---

Definitely was an overlooked copy and paste, oops.

Resolved in 80ca761639c732dde68b780ce9161bfa1da80d80.